### DISCUSSION TITLE
How are new hosts detected?

### DISCUSSION THREAD

**Discussion by wedcanto** [aceberg/WatchYourLAN#119] [Q&A] [answered]
Are new hosts detected by ping only? Or is it possible to specify specific tcp/udp ports to scan? For example 22, 135, 3389, etc.

**Comment by aceberg** [comment, marked as answer]
Not ping, hosts are detected by arp-scan.
If you need to monitor specific ports, may be this app will help:
https://github.com/aceberg/WatchYourPorts
